src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/InitiationTaskHandlerTest.java

```diff
@@ -26,37 +30,44 @@
 @ExtendWith(MockitoExtension.class)
 class InitiationTaskHandlerTest {
 
+    public static final String FIXED_DATE = "2020-12-08T15:53:36.530377";
     @Mock
     private WorkflowApiClientToInitiateTask apiClientToInitiateTask;
 
+    @Captor
+    private ArgumentCaptor<SendMessageRequest<InitiateTaskSendMessageRequest>> captor;
+
     @InjectMocks
     private InitiationTaskHandler handlerService;
 
+    private final EventInformation eventInformation = EventInformation.builder()
+        .eventId("submitAppeal")
+        .newStateId("")
+        .jurisdictionId("IA")
+        .caseTypeId("Asylum")
+        .caseReference("some case reference")
+        .dateTime(LocalDateTime.parse(FIXED_DATE))
+        .build();
+
     @Test
     void evaluateDmn() {
 
+        EvaluateDmnRequest<InitiateTaskEvaluateDmnRequest> requestParameters =
+            InitiateTaskHelper.buildInitiateTaskDmnRequest();
+
         Mockito.when(apiClientToInitiateTask.evaluateDmn(
             "getTask_IA_Asylum",
-            InitiateTaskHelper.buildInitiateTaskDmnRequest()
-        ))
-            .thenReturn(new EvaluateDmnResponse<>(Collections.emptyList()));
-
-        EventInformation eventInformation = EventInformation.builder()
-            .eventId("submitAppeal")
-            .newStateId("")
-            .jurisdictionId("IA")
-            .caseTypeId("Asylum")
-            .build();
+            requestParameters
+        )).thenReturn(new EvaluateDmnResponse<>(Collections.emptyList()));
 
         handlerService.evaluateDmn(eventInformation);
 
         Mockito.verify(apiClientToInitiateTask).evaluateDmn(
             eq("getTask_IA_Asylum"),
```

Comment:
these should now be wa-task-initiation-ia-asylum, it's also case sensitive

Reply:
done